README.md

```diff
@@ -62,7 +67,15 @@ If you copy the protocol.json file from the configs directory of this repo and r
 ],
 
 Change each occurrence of 127.0.0.1 to the IP of the system or vm running your docker image.
+If you use docker, to find the name of your machine, type :
+
+    docker-machine ls
+
+And get the ip with :
 
+    docker-machine ip "Nameofyourmachine"
+
+(By default, the machine name is "default").
```

Comment:
These steps are only useful for windows for Windows Home users who don't have hyper-v (so use docker-machine).  It should be put into a sub-section to make that distinction clear